Extensions/TileMap/JsExtension.js

```diff
@@ -1905,6 +1905,15 @@ module.exports = {
           padding: 5,
         })
       );
+      _placeholderTextNoAtlasPixiObject = new PIXI.Text(
+        'Set up an atlas image\nin the tilemap object.',
+        new PIXI.TextStyle({
+          fontFamily: 'Arial',
+          fontSize: 16,
+          align: 'center',
+          padding: 5,
+        })
+      );
```

Comment:
I wonder if you should change the text of 1 `PIXI.Text` instead of creating one for each message.
Also, should the placeholders only be instanciated if you actually need them to reduce memory footprint?

Reply:
Yes good idea, it was not optimal, I use only one text now.
Regarding the instanciation, I'd rather do pixi container manipulation in the constructor only and then only switch visibility but if you think it's important, I can change this

Reply:
I think that's it indeed safer to:
- instantiate what you need (even things that will be hidden) in the constructor
- then only switch visibility or text.